src/ui/components/Icon/styles.scss

```diff
@@ -44,6 +44,16 @@
   width: 112px;
 }
 
+.Icon-triangle-down {
+  background: url("./triangle-down.svg") center no-repeat;
+  background-size: contain;
+}
+
+.Icon-triangle-down-black {
+  background: url("./triangle-down-black.svg") center no-repeat;
```

Comment:
Yeah I bet after rebasing this was messed up, and I should use the mixin anyway but forgot even after it was pointed out, sorry.